potassium/potassium.py

```diff
@@ -224,7 +224,17 @@ def handle(path):
 
             endpoint = self._endpoints[route]
             return self._handle_generic(endpoint, request)
+        
+        @flask_app.route('/_k/warmup', methods=["GET","POST"])
```

Comment:
what's the purpose of allowing GET here? AFAICT it would never be used right? 

My leaning would just to have the POST endpoint since mutations on GET are a little against convention anyway.

Reply:
Can we also add `/_k/status` as an alias for `__status__ `? might be a good idea to put all our custom endpoints under /_k/* in the future

Reply:
I actually did add `/_k/status` 👍 just a few lines down